### PR TITLE
#396 Change variable names to camel case

### PR DESCRIPTION
**Pull request by achichikalova**
## Proposed changes
Closes issue #396.
Change variable names to camel case.

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?

### CODE REVIEW

File: webapp/src/components/Meeting.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ const Meeting = ({ meetingId }: MeetingProps) => {
     return null;
   }
   const currentParticipantId = meeting.participants.find(
-    ({ principal_id }) => principal_id === principalId
+    ({ principal_id: id }) => id === principalId
```
Comment (sheacronin) [inline webapp/src/components/Meeting.tsx:47]: This taught me about renaming a destructured variable. Thanks!

**Review by sheacronin — APPROVED** [review]
It looks like all snake case variables are corrected to use camel case now.

File: api/src/services/__tests__/competenciesService.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ describe('competenciesService', () => {
       mockQuery(
         'insert into `model_competencies` (`competency_id`, `principal_id`) values (?, ?)',
         [competencyId, principalId],
-        [model_competency_id]
+        [modelCompetencyId]
```
Comment (Nazanin-Rezaei) [inline api/src/services/__tests__/competenciesService.ts:60]: The format of this variable changes to the camel case as of the previous one.

**Review by Nazanin-Rezaei — APPROVED** [review]

**Review by MahoMori — APPROVED** [review]
All the snail case are changed to camel case which resolves this issue. It looks good to merge!

**Pull request merged**